pkg/graph/schema.graphql

```diff
@@ -61,10 +61,13 @@ enum AccessibilityRequestDocumentStatus {
 A document that belongs to an accessibility request
 """
 type AccessibilityRequestDocument {
+  bucket: String!
   id: UUID!
-  mimetype: String!
+  key: String!
+  mimeType: String!
   name: String!
```

Comment:
So, this name is supposed to be like "Testing VPAT" or "My custom doc name", but not necessarily the name of the file, like "testing_vpat.pdf". Is there a way we can set the schema to be less ambiguous? Or are you thinking we could use a combo of type + type_description (in the case of "other") for the thing I was perviously calling name?

Reply:
Oh yeah, this is intended to store the actual file name. I assumed we would add another field for the file type (and the other value) in the next story.

Reply:
Like I know there isn't strictly a need for it in the UI, but I think we're going to regret it in the future if we don't store the original filename and file size in the database.